CHANGELOG.md

```diff
@@ -68,7 +68,7 @@ COMPONENTS:
 | tre-service-guacamole-import-reviewvm | 0.2.0 |
 | tre-service-guacamole-linuxvm | 0.6.1 |
 | tre-service-guacamole-windowsvm | 0.6.0 |
-| tre-workspace-service-health | 4.1.9 |
+| tre-workspace-service-health | 0.1.0 |
```

Comment:
```suggestion
| tre-workspace-service-health | 0.2.0 |
```

Reply:
It is the fix for the previous release, you said I should add it in this PR